ballerina/Ballerina.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 org = "ballerina"
 name = "observe"
-version = "1.0.7"
+version = "1.0.8"
 distribution = "slbeta3"
 export = ["observe", "observe.mockextension" ]
 
 [[platform.java11.dependency]]
-path = "../native/build/libs/observe-native-1.0.7.jar"
+path = "../native/build/libs/observe-native-1.0.8-SNAPSHOT.jar"
```

Comment:
Shouldn't we need to add a time-stamp version here?

Reply:
These changes are auto-generated when building the observe module according to the version information in `gradle.properties` file. Commit: 7e40001e7ac0595230ca495a8085211f2319159f. I think it is okay to keep them as this is usually done in stdlib modules also.